tests/vaccination_validations.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { test, expect, Page } from "@playwright/test";
+
+let p: Page;
+
+test("Vaccination validations", async ({ page }) => {
+  p = page;
+  await given_the_app_is_setup();
+
+  await given_i_am_on_the_vaccination_page_for_a_child();
+  await when_i_continue_without_entering_the_vaccination_status();
+  await then_the_vaccination_validation_errors_are_displayed();
+
+  await given_i_select_that_they_got_the_vaccine();
+  await when_i_continue_without_specifying_a_vaccination_site();
+  await then_i_see_the_vaccination_site_validation_errors();
+
+  await given_i_move_on_to_the_how_the_vaccine_was_given_page();
+  await when_i_continue_without_entering_it_was_given();
+  await then_i_see_the_how_the_vaccine_was_given_validation_errors();
+
+  await given_i_move_on_to_the_batch_selection_page();
+  await when_i_continue_without_entering_a_batch();
+  await then_i_see_the_batch_validation_errors();
+});
+
+async function given_the_app_is_setup() {
+  await p.goto("/reset");
+}
+
+async function given_i_am_on_the_vaccination_page_for_a_child() {
+  await p.goto("/sessions/1/vaccinations");
+  await p.getByRole("tab", { name: "Action needed" }).click();
+  await p.getByRole("link", { name: "Ernie Funk" }).click();
+}
+
+async function when_i_continue_without_entering_the_vaccination_status() {
+  await p.getByRole("button", { name: "Continue" }).click();
+}
+const when_i_continue_without_specifying_a_vaccination_site =
+  when_i_continue_without_entering_the_vaccination_status;
+const when_i_continue_without_entering_it_was_given =
+  when_i_continue_without_entering_the_vaccination_status;
+const when_i_continue_without_entering_a_batch =
+  when_i_continue_without_entering_the_vaccination_status;
```

Comment:
Not sure if we need these... they're effectively comments but I feel like "when I continue without entering anything" would work just as well without the need for aliases

Reply:
Ya, I think I agree. I think I did this earlier when I didn't have as good names for the steps that came before and after, but I improved those and, ya, that ugliness isn't needed anymore.